hazelcast/src/test/resources/hazelcast-fullconfig-advanced-network-config.xml

```diff
@@ -36,7 +36,7 @@
 <hazelcast xmlns="http://www.hazelcast.com/schema/config"
            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
            xsi:schemaLocation="http://www.hazelcast.com/schema/config
-           http://www.hazelcast.com/schema/config/hazelcast-config-5.3.xsd">
+           http://www.hazelcast.com/schema/config/hazelcast-config-5.4.xsd">
```

Comment:
It seems we didn't do the usual update to next minor version, @vbekiaris should we do that in separate PR for all configs?